src/script/descriptor.h

```diff
@@ -13,6 +13,45 @@
 
 #include <vector>
 
+using ExtPubKeyMap = std::unordered_map<uint32_t, CExtPubKey>;
+
+// Cache for single descriptor's derived extended pubkeys
+class DescriptorCache {
+private:
+    std::unordered_map<uint32_t, ExtPubKeyMap> m_derived_xpubs; // Map key expression index -> map of (key derivation index -> xpub)
```

Comment:
Done

Reply:
It needs to be on the line above, otherwise this happens:
<img width="808" alt="Schermafbeelding 2020-03-07 om 15 53 36" src="https://user-images.githubusercontent.com/10217/76145663-d47e7780-608b-11ea-8f09-8d4399a0d315.png">

Reply:
Fixed